docs/concepts/models/external_models.md

```diff
@@ -42,6 +42,45 @@ All of a SQLMesh project's external models are defined in a single `external_mod
 
 Alternatively, additional external models can also be defined in the [external_models/](#using-the-external_models-directory) folder.
 
+### Using CLI
```

Comment:
Note: I moved this section up because there were references to using the CLI before the section on the CLI was introduced